client/webpack.config.node.js

```diff
@@ -159,6 +159,7 @@ const webpackConfig = {
 			} ),
 		new webpack.ExternalsPlugin( 'commonjs', getExternals() ),
 		new webpack.DefinePlugin( {
+			'typeof window': JSON.stringify( 'undefined' ),
```

Comment:
Is there a good reason to use `JSON.stringify()` on a string literal? I'd just go with the literal:

```suggestion
			'typeof window': 'undefined',
```

Reply:
I think we need to make it a string with quotes: `'"undefined"'`. That's what `JSON.stringify` is doing. It's true that we can easily achieve the goal even without it, but the surrounding code already uses it anyway.

Reply:
Ah, got it! Thanks, let's move forward then.